roles/zuul/templates/docker-compose.yaml.j2

```diff
@@ -2,11 +2,11 @@ version: '3'
 services:
   mariadb:
     container_name: "{{ container_name.mariadb }}"
-    image: {{ image_list.mariadb }}
+    image: "{{ mariadb_image }}"
```

Comment:
Why are you adding all these quotes? Seems to be working fine without them.

Reply:
It does. But all other repos use quotes, too. I just tried to keep it clean ^^